rmgpy/solver/surfaceTest.py

```diff
@@ -331,6 +331,7 @@ def test_solve_h2_coverage_dependence(self):
             initial_surface_coverages={x: 1.0},
             surface_volume_ratio=(1e1, 'm^-1'),
             surface_site_density=(2.72e-9, 'mol/cm^2'),
+            coverage_dependence=True,
```

Comment:
As noted in the commit message, if this was needed (I think it is?) then why didn't that show up in the unit test? Are we actually testing that coverage dependence works?